src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const str = sourceString.split(';');
+
+  for (const item of str) {
+    const idx = item.indexOf(':');
+    const property = item.slice(0, idx).trim();
+
+    if (property) {
+      result[property] = item.slice(idx + 1).trim();
+    }
```

Comment:
you can try to simplify this logic and split the item by ":"

Reply:
Me sims this logic isn't difficult. Could you please clarify what I need to change for the right solution?

Reply:
Is that what you mean?

Reply:
It looks better, right?